ansible/roles/kafka/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+apache_mirror: apache.mirror.anlx.net
+kafka_version: 0.8.2.2
+kafka_user: kafka
+kafka_group: kafka
+kafka_data_dir: /opt/data/kafka/data
```

Comment:
putting on the encrypted drive for now, though i don't think is strictly necessary since only metadata is planned to pass through